docs/guides/publishing.md

```diff
@@ -0,0 +1,86 @@
+# Publishing capability
+
+Once your capability is tested and works, consider publishing it in the Superface registry. This will simplify the reuse of profile, map, and/or provider in other projects. The published capabilities will be also usable by other developers.
+
+<!-- TODO: Note about private capabilities / registries -->
+
+## Prerequisites
+
+To publish content to the Superface registry, you need to:
+
+1. Have a user account at [Superface registry](https://superface.ai/).
+2. Have the [Superface CLI installed](./setup-the-environment.md).
+3. Have the local profile, map, and/or provider linked in `super.json` (as described in previous steps).
+
+Next follow these steps.
+
+## Log in to the registry
+
+First login with Superface CLI:
+
+```shell
+superface login
+```
+
+This will prompt you to open a browser or, if you decline, provides a link you can paste to a browser. Once you confirm the login return back to the terminal. You may need to optionally sign into the user account if you weren't signed in previously.
+
+You can confirm the login process was successful with `whoami` command:
+
+```shell
+superface whoami
+```
+
+## Publish the content
+
+Use the `publish` command with `profile`, `provider`, or `map`. You always need to specify a profile ID and provider name. The command has the following format"
```

Comment:
> You always need to specify a profile ID and provider name. The command has the following format"

Are we sure about this? I guess this comes from [the CLI readme](https://github.com/superfaceai/cli#superface-publish-documenttype) but AFAIK that is automatically generated by the CLI framework.

It doesn't make sense to have to specify provider name when publishing the profile. Can we please re-check these with @Jakub-Vacek ?

Reply:
You need to specify `providerName` when publishing profile to be able to access map. Map is needed to be able to lint/check before actually publishing. We could do that without providerName but we wouldn't be able to check/lint specific profile-map combinations, we would have to check/lint every profile-map combination in super.json.

Reply:
OK, I will keep it in the docs like this for now.